src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px 50px;
```

Comment:
you can add padding: 0 50px to `container ` class and remove margins left and right for `logo `and `nav`